### PR TITLE
imx8mm-var-dart: add ap20-pt1-5 support

### PR DESCRIPTION
**Pull request by max186**
Add the support for AP20-PT1-5 including the new Leica board index "D"

Signed-off-by: Massimo Toscanelli <massimo.toscanelli@leica-geosystems.com>

### CODE REVIEW

File: board/variscite/imx8mm_var_dart/imx8mm_var_dart.c [hunk 1]
```diff
@@ -275,6 +275,9 @@ int board_late_init(void)
 			case 'A':
 				env_set("leica_board", "AP20-PT1");
 				break;
+			case 'D':
```
Comment (htho-lgs) [inline board/variscite/imx8mm_var_dart/imx8mm_var_dart.c:280]: Please add case 'B' and 'C' to make it clear that these belong to "AP20-PT1"

**Review by htho-lgs — CHANGES_REQUESTED** [review]

File: board/variscite/imx8mm_var_dart/imx8mm_var_dart.c (hunk 1 above)
Comment (htho-lgs) [reply board/variscite/imx8mm_var_dart/imx8mm_var_dart.c:280]:
```suggestion
			case 'A':
			case 'B':
			case 'C':
			    env_set("leica_board", "AP20-PT1");
			    break;
			case 'D':
			...
```

Comment (max186) [reply board/variscite/imx8mm_var_dart/imx8mm_var_dart.c:280]: @htho-lgs So A, B and C need to be set as AP20-PT1?

Comment (htho-lgs) [reply board/variscite/imx8mm_var_dart/imx8mm_var_dart.c:280]: yes

**Comment by max186** [comment]
V2:
- Added cases for board id B and C

File: board/variscite/imx8mm_var_dart/imx8mm_var_dart.c [hunk 2]
```diff
@@ -275,6 +275,15 @@ int board_late_init(void)
 			case 'A':
 				env_set("leica_board", "AP20-PT1");
 				break;
+			case 'B':
+				env_set("leica_board", "AP20-PT1");
+				break;
+			case 'C':
+				env_set("leica_board", "AP20-PT1");
+				break;
```
Comment (htho-lgs) [inline board/variscite/imx8mm_var_dart/imx8mm_var_dart.c]: Why not:
```
case 'A':
case 'B':
case 'C':
    env_set("leica_board", "AP20-PT1");
    break;
```
?

**Review by htho-lgs — CHANGES_REQUESTED** [review]

File: board/variscite/imx8mm_var_dart/imx8mm_var_dart.c (hunk 2 above)
Comment (max186) [reply board/variscite/imx8mm_var_dart/imx8mm_var_dart.c]: Got it

**Review by htho-lgs — APPROVED** [review]

**Comment by max186** [comment]
V3:
- simplified cases for board id B and C

**Pull request merged**